pkg/reconciler/testing/logger.go

```diff
@@ -51,3 +61,13 @@ func setupFakeContextWithLabelKey(t zaptest.TestingT) (context.Context, context.
 	ctx, is := injection.Fake.SetupInformers(ctx, &rest.Config{})
 	return ctx, c, is
 }
+
+// setupDefaultContextWithLabelKey sets up the the Context and the default informers for the tests
+// The provided context includes the FilteredInformerFactory LabelKey.
+func setupDefaultContextWithLabelKey(t zaptest.TestingT) (context.Context, context.CancelFunc, []controller.Informer) {
```

Comment:
just wondering how `setupDefaultContextWithLabelKey` is different from `setupFakeContextWithLabelKey`? it looks same 🤔 

https://github.com/tektoncd/pipeline/blob/00b14d78ad73c055451f2a714387cd03f6222b89/pkg/reconciler/testing/logger.go#L47-L53

Reply:
Clean up how the functions are called but the difference is the  `injection.Fake` to `injection.Default`. This is needed so that the fakeworkloadAPI can be called for unit testing the spire api within `spire_test.go`.